recipes/lpsolve/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "lpsolve" %}
+{% set version = "5.5.2.5" %}
+{% set sha256 = "201a7c62b8b3360c884ee2a73ed7667e5716fc1e809755053b398c2f5b0cf28a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://downloads.sourceforge.net/project/lpsolve/lpsolve/{{ version }}/lp_solve_{{ version }}_source.tar.gz
+  sha256: {{ sha256 }}
+  
+  patches:
+      - lpsolve.patch
+
+build:
+  number: 0
```

Comment:
add 

```yaml
  skip: True  # [win]
```
to this `build:` section so that we don't produce empty windows packages.